### PR TITLE
Adding Assignmnet 2.1 and Assignment 2.2

### PR DESCRIPTION
**Pull request by Kavish15**

### CODE REVIEW

File: Week-2/Exercise 2.1/base.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bikester</title>
+    <link rel="icon" type="image/png" sizes="64x64" href="favicon.ico" />
+    <link rel="stylesheet" href="styl.css" />
+
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
Comment (jesusverma) [inline Week-2/Exercise 2.1/base.html:13]: what is the use of this if https://fonts.googleapis.com is already there

File: Week-2/Exercise 2.1/base.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            rel="icon"
+            type="image/png"
+            src="Image/youtube.png"
+            class="img-foot-one hov-two"
+          />
+           <img
+            rel="icon"
+            type="image/png"
+            src="Image/twitter.png"
+            class="img-foot-one hov-three"
```
Comment (jesusverma) [inline Week-2/Exercise 2.1/base.html:160]: does this code  has implementation of responsiveness and layout like horizontal or vertical

File: Week-2/Exercise 2.1/base.html (hunk 1 above)
Comment (Kavish15) [reply Week-2/Exercise 2.1/base.html:13]: Yes Jesus , Those are Extra Line ,  We can Comment them.

File: Week-2/Exercise 2.1/base.html (hunk 2 above)
Comment (Kavish15) [reply Week-2/Exercise 2.1/base.html:160]: No , The Footer Section Does Not have Resposiveness.

**Pull request merged**